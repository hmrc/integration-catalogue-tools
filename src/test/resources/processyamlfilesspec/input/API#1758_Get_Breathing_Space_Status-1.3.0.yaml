openapi: 3.0.3
info:
  title: API#1758 Get Breathing Space
  description: >
    HMRC are responding to new legislation being introduced in May 2021 to
    encourage people  with ‘problem debt’ to seek professional debt advice and
    resolve their financial situation. Breathing Space grants all entrants into
    the scheme a 60-day ‘breathing space’ from Creditors This API performs a
    retrieval of breathing space status.
  version: 1.2.1
  contact:
    name: Andy Coyne
servers:
  - url: https://{hostname}:{port}
    description: Different hostnames and ports are used in different IF environments
    variables:
      hostname:
        default: localhost
        description: The hostname for one of IST, Clone, and Production
      port:
        default: '443'
        description: The port for one of IST, Clone, and Production
