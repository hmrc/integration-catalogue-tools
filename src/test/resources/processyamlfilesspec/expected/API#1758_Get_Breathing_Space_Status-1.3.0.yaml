openapi: 3.0.3
info:
  title: API#1758 Get Breathing Space
  description: >
    HMRC are responding to new legislation being introduced in May 2021 to
    encourage people  with ‘problem debt’ to seek professional debt advice and
    resolve their financial situation. Breathing Space grants all entrants into
    the scheme a 60-day ‘breathing space’ from Creditors This API performs a
    retrieval of breathing space status.
  version: 1.2.1
  contact:
    name: Andy Coyne
  x-integration-catalogue:
    reviewed-date: 2022-04-22T20:27:05Z
    platform: CORE_IF
    publisher-reference: 1758
servers:
  - url: https://{hostname}:{port}
    description: Different hostnames and ports are used in different IF environments
    variables:
      hostname:
        default: localhost
        description: The hostname for one of IST, Clone, and Production
      port:
        default: '443'
        description: The port for one of IST, Clone, and Production
